package.json

```diff
@@ -47,13 +45,8 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.1.3"
   },
-  "config": {
-    "commitizen": {
-      "path": "cz-conventional-changelog"
-    }
-  },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=14.0.0"
```

Comment:
when generating a plugin with `plugin-dev` it runs `yarn install` and dev-scripts updates this in the template. Usually dev-scripts ask you to run `yarn install` again to finish initialization but the generator just continues with the execution, leaving the git hooks created without running `yarn husky install` so they don't work.